internal/agent/enrollment.go

```diff
@@ -136,6 +136,7 @@ func (a *DeviceAgent) ensureEnrollmentClient() error {
 				TLSClientConfig: &tls.Config{
 					RootCAs:      a.caCertPool,
 					Certificates: []tls.Certificate{*a.enrollmentClientCert},
+					MinVersion:   tls.VersionTLS12,
```

Comment:
The server uses a min TLS1.3: https://github.com/flightctl/flightctl/blob/49a37ef878677399c7aa6ff7475bddf08156c968/internal/crypto/tls.go#L32